modules/patterns/patterns.resolver.js

```diff
@@ -0,0 +1,13 @@
+const Pattern = require('./patterns.service');
```

Comment:
Use name `patternService` instead and rename `patterns.service` to `pattern,service`

Reply:
fixed